scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -113,21 +48,18 @@ class ThemeAggregator {
    * @returns {array} - An array of aggregated theme files
    */
   static aggregateThemes(options) {
-    if (!ThemeAggregator.validate(options)) {
-      return null;
-    }
+    if (!ThemeAggregator.validate(options)) return null;
 
-    // Create generated-themes directory.
```

Comment:
Should this comment be removed?

Reply:
Inserted a comment backhttps://github.com/cerner/terra-toolkit/pull/376/commits/4c80666bf163c0a3509a771e3ce01ea969f1bb33#diff-1672fd55492cdce050f5aa314b17029bR53